### PR TITLE
New static pages

### PR DESCRIPTION
**Pull request by clarajeanne**
3 new static pages: /jobs, /etat (etat für grosse recherchen, geschichten...), /etiquette (aka netiquette for discussions) and updated /imprint with some new links to user-profiles.

### CODE REVIEW

File: lib/translations.json [hunk 1]
```diff
@@ -342,6 +342,42 @@
       "key": "crew/lead",
       "value": "Zur Crew gehören:"
     },
+    {
+      "key": "jobs/pageTitle",
+      "value": "Jobs – Republik"
+    },
+    {
+      "key": "jobs/title",
+      "value": "Jobs  "
```
Comment (uxengine) [inline lib/translations.json]: nit: let's get rid of the trailing space in the spreadsheet and here

File: lib/translations.json [hunk 2]
```diff
@@ -342,6 +342,42 @@
...
+      "key": "etiquette/pageTitle",
+      "value": "Etiquette – Republik"
+    },
+    {
+      "key": "etiquette/title",
+      "value": "Die Etiquette"
+    },
+    {
+      "key": "etiquette/description",
+      "value": "Wie wir in der Republik debattieren\n"
```
Comment (uxengine) [inline lib/translations.json]: nit: let's get rid of the trailing line-break in the spreadsheet and here

File: lib/translations.json [hunk 3]
```diff
@@ -342,6 +342,42 @@
...
+      "key": "etat/pageTitle",
+      "value": "Etat für grosse Geschichten"
+    },
+    {
+      "key": "etat/title",
+      "value": "Etat für grosse Geschichten"
+    },
+    {
+      "key": "etat/description",
+      "value": "Erzählen Sie Ihre besten Geschichten bei uns. Dank der Grosszügigkeit unserer Verlegerinnen und Verleger können wir 2018 eine Viertelmillion Franken für aussergewöhnlichen Journalismus vergeben.\n\n"
```
Comment (uxengine) [inline lib/translations.json]: nit: let's get rid of the trailing line-breaks in the spreadsheet and here

File: pages/etat.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+      image: `${STATIC_BASE_URL}/static/team/bern.jpg`,
+      url: `${PUBLIC_BASE_URL}${url.pathname}`
+    }
+
+    return (
+      <Frame
+        url={url}
+        meta={meta}
+      >
+
```
Comment (uxengine) [inline pages/etat.js]: nit: two empty lines are too much IMHO. I'd even argue we don't need *any* empty lines to structure plain HTML/JSX markup, at least not between similar siblings like `<P>`s and `<LI>`s.

File: pages/etat.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+        <LI>Wenn alles scheitert – was haben Sie auf jeden Fall?</LI>
+
+        <LI>Was ist IHR Bezug zu diesem Thema?</LI>
+
+        <LI>Was ist das Budget, das Sie im ersten Schritt benötigen? Bitte unterscheiden Sie in Sachkosten+Spesen und Tagessätze.</LI>
+
+        <LI>Was ist der ideale Zeitplan?</LI>
+
+        <LI>Welche Unterstützung, welches Knowhow erwarten Sie von uns?</LI>
```
Comment (uxengine) [inline pages/etat.js]: Der Duden empfiehlt die Schreibweise `Know-how`, aber das Korrektorat wird wissen, was es tut. (Falsch isses nicht)

File: pages/etiquette.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+        <P>Für uns ergeben sich daraus folgende Konsequenzen:</P>
+
+        <Editorial.OL>
+
+          <LI>Beleidigen Sie einander nicht. Auch nicht, wenn das Gegenüber diese Regel schon gebrochen hat. Im Zweifelsfall: Schreiben Sie nichts, was Sie Ihrem Gegenüber nicht auch ins Gesicht sagen würden.</LI>
+
+          <LI>Sexismus, Rassismus und das Kokettieren damit hat im Wohnzimmer der Republik nichts verloren. Beiträge in diese Richtung werden nicht geduldet. Ebensowenig sämtliche Beiträge, die gegen das Gesetz verstossen.</LI>
+
+          <LI>Belästigen Sie andere Menschen nicht. Will eine Person nicht mit ihnen diskutieren, akzeptieren Sie das bitte.</LI>
```
Comment (uxengine) [inline pages/etiquette.js]: `ihnen` gross schreiben

File: pages/etat.js [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+      <P>Der Fonds für grosse Recherchen, Geschichten und Ideen kommt von der Project R Genossenschaft, dem gemeinnützigen Dach über der Republik. Project R will die Demokratie stärken, indem sie den Journalismus als vierte Gewalt weiterentwickelt. Dem Gemeinwohl verpflichtet, fördert Project R den konstruktiven Diskurs und diskriminierungsfreie, vielfältige Debatten über die Fragen unserer Zeit.</P>
+
+      <P>Entschieden wird über die Projekte ausschliesslich innerhalb der Republik-Redaktion und ausschliesslich nach journalistischen Kriterien. Wir freuen uns, von Ihnen zu hören!</P>
+
+      <P>Das Team von Republik und <A href='https://project-r.construction'>Project R</A></P>
+
+      <P>PS: Unser <A href='https://www.republik.ch/2018/01/19/usa-serie'>erster Serienprototyp</A> wurde bereits mit Mitteln aus dem Etat finanziert.</P>
```
Comment (uxengine) [inline pages/etat.js]: could be just a relative URL:
`<A href='/2018/01/19/usa-serie'>erster Serienprototyp</A>`

File: pages/jobs.js [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+        <P>Du hast Lust, bei uns viel Verantwortung zu übernehmen. Zusammen mit dem Team willst du die Republik langfristig weiterzuentwickeln und maintainen. Du geniesst viel Freiheit zu haben, bringst dich gerne auch konzeptionell ein und interessierst dich für verschiedene Aspekte unseres Produktes: von einer möglichst stabilen und einfachen Payments-Lösung über unser super cooles wysiwyg-markdown-github-CMS hin zu neuen Tools für spannende User-Interaktion.</P>
+
+        <P>Du bist interessiert? Dann bewirb dich mit einer E-Mail bei <A href='mailto:clara.vuillemin@republik.ch'>clara.vuillemin@republik.ch</A></P>
+
+        <P>Bitte beantworte in der Bewerbung – neben dem, was du uns sonst noch sagen möchtest – folgende Fragen: </P>
+
+        <Editorial.UL>
+          <LI> Was denkst du über unser Produkt? Was gefällt dir daran? Was würdest du ändern?</LI>
+          <LI>Was interessiert dich am Journalismus?</LI>
+          <LI>Was macht dich zu einer guten Coderin, einem gute Coder?</LI>
```
Comment (uxengine) [inline pages/jobs.js]: ... einem gute**n** Coder ... (mit n)

File: pages/jobs.js [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+          <LI>Hast du Projekte, auf die du besonders stolz bist? Dann freuen wir uns, über Links zu Open Source Repos, Websites etc. und eine Begründung, warum du besonders stolz darauf bist.</LI>
```
Comment (uxengine) [inline pages/jobs.js]: Dann freuen wir uns über Links ... (no comma)

File: pages/jobs.js [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+        <P>Wir suchen dich. Wir sind das Team der Republik. Wir arbeiten für relevanten, unabhängigen und innovativen Journalismus ohne Bullshit. Und wir sind eine interdisziplinäre, qualifizierte, hart arbeitende, leidenschaftliche und fröhliche <A href='/crew'>Crew</A>. Unser Code ist vollständig Open-Source: <A href='https://github.com/orbiting'>github.com/orbiting</A>.</P>
+
+        <P>Du bist eine erfahrene node.js-Entwicklerin. Insbesondere GraphQL und PostgresQL sind dir vertraut. Du hast in der Vergangenheit schon Verantwortung für Software-Projekte getragen. Und du weisst, was es braucht, ein Konzept erfolgreich umzusetzen bis hin zum laufenden Betrieb. </P>
+
+        <P>Du hast Lust, bei uns viel Verantwortung zu übernehmen. Zusammen mit dem Team willst du die Republik langfristig weiterzuentwickeln und maintainen. Du geniesst viel Freiheit zu haben, bringst dich gerne auch konzeptionell ein und interessierst dich für verschiedene Aspekte unseres Produktes: von einer möglichst stabilen und einfachen Payments-Lösung über unser super cooles wysiwyg-markdown-github-CMS hin zu neuen Tools für spannende User-Interaktion.</P>
```
Comment (uxengine) [inline pages/jobs.js]: weiterentwickeln (ohne zu)

**Comment by uxengine** [comment]
Pls consider this approved after fixing the typos.

**Comment by clarajeanne** [comment]
@tpreusse should be ok now. pls check and merge if everything is fine. btw. i put the 'etiquette' in the section 'Meine Republik', makes the footer more balanced and makes sense to me (faq is also there).

**Pull request merged**